### PR TITLE
Revert <jetty.version>11.0.13</jetty.version>

### PR DESCRIPTION
**Pull request by jepsar**
Reverts <jetty.version>11.0.13</jetty.version>

### CODE REVIEW

**Comment by fcorneli** [comment]
@jepsar you're faster than me :)

File: primefaces-integration-tests/pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
         <tomcat.version>9.0.71</tomcat.version>
         <resteasy.version>5.0.5.Final</resteasy.version>
         <owb.version>2.0.27</owb.version>
-        <jetty.version>9.4.51.v20230217</jetty.version>
```
Comment (tandraschko) [inline primefaces-integration-tests/pom.xml]: only jakarta needs to be reverted

Comment (jepsar) [reply primefaces-integration-tests/pom.xml]: Ah, I just reverted the entire PR.

File: primefaces-integration-tests/pom.xml [hunk 2]
```diff
@@ -36,7 +36,7 @@
...
+        <jetty.version>9.4.50.v20221201</jetty.version>
```
Comment (jepsar) [inline primefaces-integration-tests/pom.xml]:
```suggestion
        <jetty.version>9.4.51.v20230217</jetty.version>
```

File: primefaces-showcase/pom.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
         <myfaces.version>2.3.10</myfaces.version>
         <myfaces-next.version>2.3-next-M7</myfaces-next.version>
         <resteasy.version>5.0.5.Final</resteasy.version>
-        <jetty.version>9.4.51.v20230217</jetty.version>
+        <jetty.version>9.4.50.v20221201</jetty.version>
```
Comment (jepsar) [inline primefaces-showcase/pom.xml]:
```suggestion
        <jetty.version>9.4.51.v20230217</jetty.version>
```

**Comment by melloware** [comment]
Sorry I forgot we added the jakarta tests!  Thanks for fixing.

**Pull request merged**